File: application.client/src/DTOs/GetWorkspaces/WorkspaceDto.ts
import { WorkspaceType } from "../../models/WorkspaceType"
import { Amenity } from "../../models/Amenity"
import { BookingDto } from "./BookingDto"

export interface WorkspaceDto
{
  id: number,
  capacity: number,
  amenities: Amenity[]
  workspaceType: WorkspaceType
  bookings: BookingDto[]
}

File: Application.Server/Models/DTOs/BookingQuestion/WorkspaceDto.cs
using Application.Server.Models.CoworkingDatabase;

namespace Application.Server.Models.DTOs.BookingQuestion
{
    public class WorkspaceDto
    {
        public int Id { get; set; }
        public int Capacity { get; set; }
        public WorkspaceType WorkspaceType { get; set; }
        public CoworkingDto Coworking { get; set; } = null!;

    }
}
